crates/fluvio-test/src/tests/consumer_offsets/auto/flush.rs

```diff
@@ -45,9 +45,12 @@ pub async fn test_strategy_auto_periodic_flush(
     }
     sleep(Duration::from_secs(2)).await; // yeild  to drive auto flush flow
 
-    let consumer = find_consumer(client, &consumer_id, 0).await?;
-    ensure!(consumer.is_some());
-    ensure!(consumer.unwrap().offset > 0i64);
+    for partition in 0..partitions {
+        let offset = (counts[partition]) as i64;
+        let consumer = find_consumer(client, &consumer_id, partition).await?;
+        ensure!(consumer.is_some());
+        ensure!(consumer.unwrap().offset == offset);
```

Comment:
I tested this without these modifications, and it was falling because it was not including the last read